python/metatensor-torch/metatensor/torch/atomistic/_extensions.py

```diff
@@ -12,15 +12,24 @@
 
 
 def _rascaline_lib_path():
+    # This is kept for backward compatibility, but rascaline is now named featomic.
+    # This code should be removed by the middle of 2025.
     import rascaline
 
     return [rascaline._c_lib._lib_path()]
 
 
+def _featomic_lib_path():
+    import featomic
+
+    return [featomic._c_lib._lib_path()]
+
+
 # Manual definition of which TorchScript extensions have their own dependencies. The
 # dependencies should be returned in the order they need to be loaded.
 EXTENSIONS_WITH_DEPENDENCIES = {
     "rascaline_torch": _rascaline_lib_path,
```

Comment:
Is this part of the backward compatibility too, or should it be removed?

Reply:
Yes, the function above is only used here, so both should be kept & removed at the same time

Reply:
Ok great!